Makes folder show files uploaded individually

StevenWessinger opened the pull request:
This update is made by adding a new refresh event
which is called each time a file is uploaded. This
will help users to be confident that their files
are uploading. This is especially relevant if there
are multiple large files being uploaded at once.

StevenWessinger committed:
Makes folder show files uploaded individually

This update is made by adding a new refresh event
which is called each time a file is uploaded. This
will help users to be confident that their files
are uploading. This is especially relevant if there
are multiple large files being uploaded at once.

StevenWessinger commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Bug - S3commander: List of files should update after each file completes in multi-file upload.](https://trello.com/c/KEF1836t/4-bug-s3commander-list-of-files-should-update-after-each-file-completes-in-multi-file-upload)

kevinrobbins commented on src/dropzone/DropzoneDirective.ts:
```diff
@@ -77,6 +77,9 @@ export class DropzoneDirective implements ng.IDirective {
           this.emit('queuecomplete');
         }
       },
+      'refresh': function(file: any) {
```
In this case, I would move the onRefresh call to be within the 'success' event handler. The 'success' event handler defines the behavior that occurs when the success event is fired. Part of that behavior would involve the refreshing of the UI. An event handler called 'refresh' would indicate to me the behavior that is to occur once something has refreshed which is not what is happening here.

StevenWessinger committed:
Integrate refresh signal with success signal

This change was suggested by Kevin. The refresh signal may cause
confusion because it was not triggered when the page was refreshed.
This removes that refresh signal, and has the folder be refreshed
in the success signal instead.

mliudev commented:
Looks good. 👍
